lib/screens/explore_screens/vouch_for_a_member/interactor/viewmodel/vouch_for_a_member_page_commands.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:seeds/datasource/remote/model/member_model.dart';
+import 'package:seeds/domain-shared/page_command.dart';
+
+class ShowVouchForMemberConfirmation extends PageCommand {
+  final MemberModel selectedMember;
+
+  ShowVouchForMemberConfirmation(this.selectedMember);
```

Comment:
The selected member can be grabbed from the state

Reply:
fix